fulllogging.properties

```diff
@@ -0,0 +1,17 @@
+# Global logging properties.
```

Comment:
Here are the things I've tried:
1) Since that file, by that name, is included in the jmDNS distribution, I included it in case there's code (which doesn't show in the jmDNS source, but that's known to be not consistent with the .jar) in mDNS itself that loads it.
2) I set the property that tells Java itself to load it.
3) (current Travis run, not done yet) I've updated slf4J to current, set up the jul-to-slf4J bridge, and configured that in log4J.

Any helpful suggestions for getting right of th3 5k of log file lines?